ssh_check/manifest.json

```diff
@@ -15,9 +16,9 @@
     "network"
   ],
   "type": "check",
-  "doc_link": "https://docs.datadoghq.com/integrations/ssh_check/",
+  "metric_prefix": "sftp.", // is this valid?
```

Comment:
not sure what it should be, we submit both ssh and sftp metrics. Integration in the app uses ssh so i will keep ssh.